extensions/reclaim-ai/src/notifications.tsx

```diff
@@ -121,7 +121,18 @@ export default function Command() {
 
   const titleInfo = useMemo<TitleInfo>(() => {
     const now = new Date();
-    const eventNextNow = eventMoment?.event;
+    let eventNextNow;
+    const showNowEvent = getPreferenceValues()["showNowEvent"];
+    if (showNowEvent) {
+      const nowEvent = events?.filter((event) => {
+        const start = new Date(event.eventStart);
+        const end = new Date(event.eventEnd);
+        return isWithinInterval(now, { start, end });
+      });
+      eventNextNow = nowEvent?.[0];
+    } else {
```

Comment:
Ideally if there is no "now" event and `showNowEvent` is set, we should fallback to `eventMoment.event`